sources/gerritDB/index.js

```diff
@@ -53,11 +61,12 @@ var gerritDB = (function () {
             "draft": null
         };
 
-        var query = "SELECT COUNT(*) AS Count, status AS reviewType"
-            + " FROM changes"
-            + " GROUP BY reviewType";
+        var query =
```

Comment:
We need to move all of the query strings  to the new file. We may do later commits